base/applications/notepad/main.c

```diff
@@ -427,8 +427,13 @@ NOTEPAD_WndProc(HWND hWnd, UINT msg, WPARAM wParam, LPARAM lParam)
                                   LOWORD(wParam),
                                   SWP_NOZORDER);
 
-            if (hdwp != NULL)
-                EndDeferWindowPos(hdwp);
+            if (hdwp == NULL)
+                break;
+                
+            EndDeferWindowPos(hdwp);
+
+            /* Align status bar parts */
+            DIALOG_StatusBarAlignParts();
```

Comment:
You sure you don't want to do alignment if the hdwp operations above failed at some point?

Reply:
Yes, the alignment is done only if the resize operation of the status bar succeeds.